### ISSUE TITLE
Looking for (co-)maintainer

### ISSUE THREAD

**Issue by marzzzello** [marzzzello/pytr#72]
Hello everyone,
I wanted to share an update regarding pytr. As you may have noticed there hasn't been much development activity since a while. I initially started this project for my personal use, to automate downloading PDFs from TradeRepublic and import them into Portfolio Performance. However, I was pleasantly surprised to see that more people than I anticipated found value in this project and began using it as well. As the user base grew, so did the responsibility to maintain and improve the project.

Since I no longer use TradeRepublic, I haven't felt the need to continue improving and fixing things. However, I understand that some of you may still find this project useful and want to keep it alive.
That's why I am looking for people that are interested in maintaining and further developing this. I encourage you to reach out by commenting here or sending me an [email](mailto:pytr@07f.de). If you're already familiar with the code base and have made some contributions through pull requests, that would be fantastic (though not required). We can then discuss how to proceed.
 
In the meantime, everyone can contribute by responding to open issues, submitting new pull requests, and reviewing existing ones. When a pull request has positive feedback from others, I can merge and release new versions. Let's work together to keep this project thriving and useful for everyone involved.

**Comment by yubiuser** [comment]
Looking at the recent pull requests and interactions, @MartinScharrer and/or @Katzmann1983 might be good candidates if they are willing to take over.

**Comment by MartinScharrer** [comment]
I would have interest. Currently I am working on some larger modification to export CSVs for Portfolio Performance based on my pull request. My plan was to create a new script based on `pytr` which works a little different and publish it as a new project, but I could also port some of it to the original script.

**Comment by VerlorenerPapa** [comment]
Hello everybody! I too would have interested but for the moment I can't get any version yubiuser and marzzzello properly working. Not a good starting point to help.

**Comment by marzzzello** [comment]
@MartinScharrer Thank you, I send you a maintainer invitation

**Comment by Katzmann1983** [comment]
Thanks for the honor, @marzzzello . Would love too help here :D

**Comment by marzzzello** [comment]
@Katzmann1983 Thanks, I also send you an invitation

**Comment by Katzmann1983** [comment]
Thank you @marzzzello  
Do you also want me to be able to upload a new version to pip?

**Comment by marzzzello** [comment]
@Katzmann1983 The CI automatically publishes to PyPI on new tags. So no need to manually publish releases there

**Comment by MartinScharrer** [comment]
I had some ideas about restructing the package and adding a conversion into CSV which is compatible for direct import with *Portfolio Performance* (PP).  What I basically needed was only the dl_docs command to get the PDFs and an export into CSV for the orders and the (non-order related) cash account transactions for two separate imports.

As this was quite a big change I first started this as a branch in my pytr fork, but then decided to do a separate package. This way I was able to add some different handling and command line options without worrying about backwards compatibility. You find the package under https://github.com/martinscharrer/pytrpp/. The git repository is still based on the one of pytr but github does not handle it as fork.

We can discuss backporting some of the features into pytr.

**Comment by RealCLanger** [comment]
Hi, I can see that there has been no maintainer activity for about two months, since early November. Could anything be done about that? I would also be willing to help with maintenance.

**Comment by marzzzello** [comment]
@RealCLanger Hi, thanks for the offer. I send you an invite for maintainer.

**Comment by RealCLanger** [comment]
> [@RealCLanger](https://github.com/RealCLanger) Hi, thanks for the offer. I send you an invite for maintainer.

Thank you. Is there any guidance about releasing a new version?

**Comment by ExploracuriousAlex** [comment]
@marzzzello  It would be a shame if this useful project became outdated and bug fixes, changes, and improvements were scattered across many different forks. Therefore, I would like to apply to support as co-maintainer. Although I don't currently have the time to look at and fully understand all major changes in detail, I can at least contribute by incorporating small to medium changes that have no negative impact on the project and that I can oversee. Such as  #248

**Comment by MartinScharrer** [comment]
I just saw that TR has changed the JSON response format. The crucial `eventType` field is missing now. This makes it very difficult to process the entries. My fork `pytrpp` is quite broken now.

**Comment by RealCLanger** [comment]
> [@marzzzello](https://github.com/marzzzello) It would be a shame if this useful project became outdated and bug fixes, changes, and improvements were scattered across many different forks. Therefore, I would like to apply to support as co-maintainer. Although I don't currently have the time to look at and fully understand all major changes in detail, I can at least contribute by incorporating small to medium changes that have no negative impact on the project and that I can oversee. Such as [#248](https://github.com/pytr-org/pytr/pull/248)

I agree, it would be nice to have more people contributing to the project.

**Comment by RealCLanger** [comment]
> I just saw that TR has changed the JSON response format. The crucial `eventType` field is missing now. This makes it very difficult to process the entries. My fork `pytrpp` is quite broken now.

Yes, I'll try to come up with a new event parser that can handle the new format. However, without having an `eventType` field, it could become more heuristic.

**Comment by ExploracuriousAlex** [comment]
@marzzzello @RealCLanger Hi, would you please make me a collaborator? I’d like to support the project by helping to merge PRs and close open issues. I also suggest using additional issues to make sure we’re working in the same direction on improvements, following a common roadmap and coordinating changes.
